stories/core/Table.stories.tsx

```diff
@@ -486,6 +486,26 @@ Filters.args = {
   ],
   emptyFilteredTableContent: 'No results found. Clear or try another filter.',
 };
+Filters.parameters = {
+  creevey: {
+    tests: {
+      async open() {
+        const nameFilter = (
+          await this.browser.findElements({
+            css: '.iui-filter-button',
+          })
+        )[1];
+        await this.browser.actions().move({ origin: nameFilter }).perform();
```

Comment:
you need to move there before clicking?

Reply:
Yeah it didn't work without

Reply:
maybe stack click on the same actions then? `...move({origin: nameFilter}).click().perform()`

Reply:
I could, but that reaches the line limit, so it gets wrapped into 4 lines